GameData/RP-1/Programs/Programs.cfg

```diff
@@ -5,8 +5,8 @@ RP0_PROGRAM
 	description = This program tasks the space agency with the development of experimental aircraft to test aerodynamic behavior at velocities and altitudes previously unreachable by crewed craft.
 	objectivesPrettyText = Complete X-Planes (Mach 2 Supersonic) and X-Planes (Karman Line) contracts.
 	nominalDurationYears = 9
-	baseFunding = 600000
-	fundingCurve = BackloadedFundingCurve
+	baseFunding = 800000
```

Comment:
Using the spreadsheet, put this number / duration in the "average funds/year for normal" column and it should spit out the confidence costs and the rep (apparently we switched to symmetric rep, so use the higher number for both early-complete and late-complete).

Reply:
Done. The resulting rep gain seemed high so I reset the multiplier back to 1 (hence 130/year rather than 195). I reduced the confidence multiplier from 0.8 to 0.7--I think cutting off the fast X-Planes/fast SR makes it inviable to take as a starting program with the present curve, since both normal X-planes/breakneck SR and fast X-planes/normal SR pay less over the first two years than breakneck/fast SRs even before accounting for astronaut costs. I don't know how strongly you want to avoid playing with those multipliers--I think a viable alternative would be increasing program funding and reducing how much it's backloaded until fast X-planes/normal SR pays competitively at the start, but fussing with the curve is enough more work that I want feedback on which approach is preferable before doing so.

Reply:
Yeah I think it's fine to go that approach rather than messing further with funding or curve 👍